.pre-commit-config.yaml

```diff
@@ -10,3 +10,17 @@ repos:
     exclude: ^tests/
   - id: check-model-has-properties-file
     exclude: ^tests/
+
+- repo: https://github.com/sqlfluff/sqlfluff
+  rev: 2.1.0
+  hooks:
+  - id: sqlfluff-fix
+    args: ['--force', '--dialect', 'databricks']
+    additional_dependencies:
+      - dbt-databricks==1.2.1
```

Comment:
Do we need this dependency for linting?

Reply:
Yes, we need that. Pre-commit hooks run in a different environment than the rest of project, this is why we need add it as a additional dependency.